docs/other-topics/other-data-types.mdx

```diff
@@ -526,7 +527,7 @@ to request the addition of a new built-in DataType.
 [mssql-exact-decimals]: https://docs.microsoft.com/en-us/sql/t-sql/data-types/decimal-and-numeric-transact-sql?view=sql-server-ver15
 [mssql-inexact-decimals]: https://docs.microsoft.com/en-us/sql/t-sql/data-types/float-and-real-transact-sql?view=sql-server-ver15
 
-[sqlite-datatypes]: https://www.sqlite.org/datatype3.html
+[sqlite-datatypes]: https://www.sqlite.org/stricttables.html

```

Comment:
What is wrong with the data types document?

Reply:
`datatypes3.html` is about dynamic data types. SQLite lets your store anything in any column except if you use strict tables. To be compatible with strict tables, sequelize only uses SQLite Data Types from strict tables, so the new documentation points to that page instead